packages/gatsby-theme-api-docs/src/components/type/type.js

```diff
@@ -12,13 +12,44 @@ import Enum from './enum';
 import Properties from './properties/properties';
 import Examples from './examples';
 
+const contentTypeToPrimitiveMap = {
+  'application/json': {
+    number: '_Any JSON "number"',
+    any: '_Any valid JSON_',
+    object: '_Any JSON "object"_',
+    boolean: '_Any JSON "boolean"_',
+    string: '_Any JSON "string"_',
+    array: '_Any JSON "array"_',
+  },
+};
+
+export const getPrimitiveTypeByName = (contentType, type) =>
```

Comment:
(just an idea because it was misleading me in my understanding of the code that uses the function. It's not returning a type or type name, it's returning the description for a type if it's by chance a primitive)

```suggestion
export const getDescriptionIfPrimitiveType = (contentType, type) =>
```